### PR TITLE
Dockerfiles CI validations

### PR DESCRIPTION
**Pull request by dorschw**
## Status
In Progress

## Related Issues
Related: https://github.com/demisto/etc/issues/44846

## Description
Making sure pipfile matches its lock, and more. see issue.

### CODE REVIEW

**Comment by ilaner** [comment]
I suggest converting the `add_dependabot.sh` to python as well if we are already converting it

**Comment by dorschw** [comment]
> I suggest converting the `add_dependabot.sh` to python as well if we are already converting it

already done :) 
https://github.com/demisto/dockerfiles/blob/3be0935b299a3ecac4cf294ffe233e2178a2c8c1/test_pipfile_lock.py#L35-L69

**Review by ilaner — COMMENTED** [review]
see my comment

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,50 +1,37 @@
...
-          name: Validate Dependabot Config
-          command: ./docker/verify_dependabot.sh
       - setup_remote_docker:
           version: 20.10.6
       - add_ssh_keys:
           fingerprints:
             - 46:c2:72:91:9d:0e:2d:3c:93:d6:6c:12:31:df:8e:db
+      - run:
+          name: Validate Images and Dependabot
+          command: python -m test_pipfile_all.py
```
Comment (ilaner) [inline .circleci/config.yml:17]:
```suggestion
          command: python -m test_pipfile_all
```

**Pull request closed without merging**